filter.js

```diff
@@ -38,9 +41,9 @@ function Filter(inputTree, options) {
   this._destFilePathCache = Object.create(null);
 }
 
-Filter.prototype.rebuild = function() {
+Filter.prototype.build = function build() {
   var self = this;
-  var srcDir = this.inputPath;
+  var srcDir = this.inputPaths[0];
```

Comment:
We'll be able to add centralized error handling for this to Broccoli to catch these sorts of errors (`undefined` nodes, etc.). Now that we capture the [instantiation stack](https://github.com/broccolijs/broccoli-plugin/blob/93eda0da6a19764948b5c10c6e6ea75977edc374/index.js#L10), the reporting will also be able to display a helpful stack.